assets/js/base/components/select/index.js

```diff
@@ -32,7 +32,7 @@ const Select = ( {
 					onChange( selectedItem.key );
 				} }
 				options={ options }
-				value={ value }
+				value={ value || null }
```

Comment:
`value` might be undefined, but when it changes from undefined to a value, downshift shows a warning that the component is changing from being uncontrolled to controlled. That can be fixed setting `null` as the default value when non is selected.